Sources/SwiftSyntax/SourceLength.swift

```diff
@@ -11,64 +11,46 @@
 //===----------------------------------------------------------------------===//
 
 /// The length a syntax node spans in the source code. From any AbsolutePosition
-/// you reach a node's end location by either adding its UTF-8 length or by
-/// inserting `lines` newlines and then moving `columns` columns to the right.
-public struct SourceLength {
-  public let newlines: Int
-  public let columnsAtLastLine: Int
+/// you reach a node's end location by adding its UTF-8 length.
+public struct SourceLength: Comparable {
   public let utf8Length: Int
 
   /// Construct the source length of a given text
   public init(of text: String) {
-    var newlines = 0
-    var columnsAtLastLine = 0
-    var utf8Length = 0
-    for char in text {
-      let charLength = String(char).utf8.count
-      utf8Length += charLength
-      switch char {
-      case "\n", "\r\n", "\r":
-        newlines += 1
-        columnsAtLastLine = 0
-      default:
-        columnsAtLastLine += charLength
-      }
-    }
-    self.newlines = newlines
-    self.columnsAtLastLine = columnsAtLastLine
-    self.utf8Length = utf8Length
+    self.utf8Length = text.utf8.count
   }
 
-  public init(newlines: Int, columnsAtLastLine: Int, utf8Length: Int) {
-    self.newlines = newlines
-    self.columnsAtLastLine = columnsAtLastLine
+  public init(utf8Length: Int) {
     self.utf8Length = utf8Length
   }
 
   /// A zero-length source length
   public static let zero: SourceLength = 
-      SourceLength(newlines: 0, columnsAtLastLine: 0, utf8Length: 0)
+      SourceLength(utf8Length: 0)
+
+  public static func <(lhs: SourceLength, rhs: SourceLength) -> Bool {
+    return lhs.utf8Length < rhs.utf8Length
+  }
 
-  /// Combine the length of two source length. Note that the addition is *not*
-  /// commutative (3 columns + 1 line = 1 line but 1 line + 3 columns = 1 line
-  /// and 3 columns)
+  /// Combine the length of two source length.
   public static func +(lhs: SourceLength, rhs: SourceLength) -> SourceLength {
+    _ = lhs == rhs
```

Comment:
Seems like this is left-over from debugging

Reply:
Good catch! Removed.